config/console-plugin/kustomization.yaml

```diff
@@ -12,9 +12,4 @@ resources:
 # - ../../patches/common
 
 generatorOptions:
```

Comment:
This one I think we want to try locally as it might break our existing manual build workflow? @darkdoc wdyt?